### PR TITLE
feat: tslint-config extend tslint-config-eslint/sonar

### PR DESCRIPTION
**Pull request by EdisonSu768**

### CODE REVIEW

File: packages/tslint-config/eslint.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
   "$schema": "http://json.schemastore.org/tslint",
   "extends": [
     "./tslint.json",
-    "tslint-config-eslint/prettier"
+    "tslint-config-eslint/prettier",
+    "tslint-config-eslint/sonar"
   ],
   "rules": {
```
Comment (JounQin) [inline packages/tslint-config/eslint.json]: These rules can be removed then.

Comment (EdisonSu768) [reply packages/tslint-config/eslint.json]: so, should we add this rule directly in rules, and remove prettier and sonar's extends ?

Comment (JounQin) [reply packages/tslint-config/eslint.json]: I prefer:

```jsonc
{
  "$schema": "http://json.schemastore.org/tslint",
  "extends": [
    "./tslint.json",
    "tslint-config-eslint/sonar",
    "tslint-config-eslint/prettier"
  ]
}
```

`prettier` should always take highest priority.

Comment (EdisonSu768) [reply packages/tslint-config/eslint.json]: sorry i forgot the priority

Comment (JounQin) [reply packages/tslint-config/eslint.json]: ? Why we're not removing `rules`?

Comment (EdisonSu768) [reply packages/tslint-config/eslint.json]: updated this PR

**Pull request merged**